apps/onnx/model.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import absolute_import
```

Comment:
FWIW, we just switched ~all the code in Halide to assume/require Py3; is this still necessary?

Reply:
This is only necessary with python v2. I've removed the imports.